Lokad.ILPack/src/AssemblyGenerator.Attributes.cs

```diff
@@ -20,10 +20,7 @@ public void CreateCustomAttributes(EntityHandle parent, IEnumerable<CustomAttrib
                 GetOrCreateType(type); // create type
 
                 var args = attr.ConstructorArguments;
-                var text = string.Join(",", args.Select(x =>
-                {
-                    return $"\"{x.Value}";
-                }));
+                var text = string.Join(",", args.Select(x => { return $"\"{x.Value}"; }));
```

Comment:
Cosmetic, you can probably write `args.Select(x => $"\"{x.Value}")`.

Reply:
I'll make sure to include it in the next commits 👍